client/src/utils/hooks/useScreenSize.ts

```diff
@@ -28,7 +29,56 @@ function useScreenSize() {
     updateSize();
     return () => window.removeEventListener('resize', updateSize);
   }, [ss]);
-  return ss;
+
+  const down = (target: ScreenSize) => {
+    switch (target) {
+      case ScreenSize.XS:
+        return ss === ScreenSize.XS;
+      case ScreenSize.SM:
+        return ss === ScreenSize.XS || ss === ScreenSize.SM;
+      case ScreenSize.MD:
+        return (
+          ss === ScreenSize.XS || ss === ScreenSize.SM || ss === ScreenSize.MD
+        );
+      case ScreenSize.LG:
+        return (
+          ss === ScreenSize.XS ||
+          ss === ScreenSize.SM ||
```

Comment:
We could do something like `ss !== ScreenSize.XL` here, but I was going for readability here.
Thoughts?